docs/guide/breakpoint-preview.md

```diff
@@ -0,0 +1,159 @@
+
+## Device breakpoint preview
+
+Pages can be enabled to emulate devices, like phones and tablets, making it easy to see how your content will look at different screen sizes.
```

Comment:
"Pages can be enabled to..." is awkward. How about "Apostrophe provides a convenient way to emulate devices..."

Reply:
Changed.